examples/todoapp/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: todoapp
 description: A new Flutter project.
 homepage: https://primeholding.com
-version: 1.0.0+1
+version: 1.0.20+80
```

Comment:
Please avoid such changes. The version should be controlled by the tag name

Reply:
Fixed